src/replica/mysql/DatabaseMySQLGenerator.h

```diff
@@ -402,6 +403,33 @@ class QueryGenerator {
     std::string insertPacked(std::string const& tableName, std::string const& packedColumns,
                              std::vector<std::string> const& packedValues) const;
 
+    /**
+     * Generate a collection of complete INSERT statements for the given input, where
+     * the maximum size of each query string is determined by a value of the parameter
+     * \param maxQueryLength.
```

Comment:
Is that supposed to be @param? "the parmaeter \param maxquerylength." I'm don't think that's what you wanted.

Reply:
My bad. I assumed I could use this syntax to generate a reference to the parameter. After re-reading the documentation on Doxygen that's not possible...and not needed here. Thank you for noticing this!